asset/compressed_file_reader/index.js

```diff
@@ -191,15 +192,21 @@ function schema() {
             format: String,
         },
         format: {
-            doc: 'How to parse: `json` is newline-delimited json, `pass` leaves parsing for downstream operations.',
-            default: 'pass',
-            format: ['pass', 'json']
+            doc: 'How to parse: `json` is newline-delimited json, `raw` leaves parsing for downstream operations.',
+            default: 'raw',
+            format: ['raw', 'json']
         },
         size: {
             doc: 'Determines slice size in bytes',
             default: 100000,
             format: Number,
         },
+        _dead_letter_action: {
```

Comment:
This is part of the default operation schema, so you don't need to set this here.

Reply:
Remove this from the schema check and bumped minor version